### PR TITLE
Add new PowerShell book

### PR DESCRIPTION
**Pull request by TravisEz13**
Hi, I thought that `Learn PowerShell in a Month of Lunches, Linux and macOS Edition` might be great addition to your list of books. 
Thank you for your consideration

### CODE REVIEW

**Comment by TravisEz13** [comment]
ping @nikitavoloboev

**Comment by nikitavoloboev** [comment]
Address the points please

File: README.md [hunk 1]
```diff
@@ -849,6 +849,10 @@ The Best Interface Is No Interface: The Simple Path to Brilliant Technology
 
 - [📖 Modern Perl (2016)](http://modernperlbooks.com/books/modern_perl_2016/index.html)
 
+### PowerShell
+
+- [📖 Learn PowerShell in a Month of Lunches, Linux and macOS Edition (2019)](https://www.manning.com/books/learn-powershell-in-a-month-of-lunches-linux-and-macos-edition)
```
[the remark this answers is not in the document]
Comment (TravisEz13) [reply README.md]: The book is not free.

[the remark this answers is not in the document]
Comment (nikitavoloboev) [reply README.md]: Then change emoji

**Pull request merged**